baseplate/clients/redis.py

```diff
@@ -26,10 +26,10 @@ def pool_from_config(
 
     * ``url`` (required): a URL like ``redis://localhost/0``.
     * ``max_connections``: an integer maximum number of connections in the pool
-    * ``socket_connect_timeout``: a timespan of how long to wait for sockets
-        to connect. e.g. ``200 milliseconds``.
-    * ``socket_timeout``: a timespan of how long to wait for socket operations,
-        e.g. ``200 milliseconds``.
+    * ``socket_connect_timeout``: how long to wait for sockets to connect. e.g.
```

Comment:
This may be answered in one of the other files touched in this PR, but I'm not sure what valid input looks like for these timeout kwargs. The example is `200 milliseconds`, so it seems to be a string with the unit name as the second token. Would it stand to reason that I could also do `0.2 seconds`? If I just pass an int, does that work? If so, what does that mean?

This might be a more general question for any other kwarg/param that accepts relative times, but I am not yet sure of whether we have others that fit this description.

Reply:
It's one of these: https://spladug.s3.amazonaws.com/baseplate/api/baseplate/lib/config.html#baseplate.lib.config.Timespan

What would be a good way to link back to that?

Reply:
Took a stab at just linking this from the various docs for `_from_config` functions